src/System.Linq/tests/ConsistencyTests.cs

```diff
@@ -22,16 +22,18 @@ public static void MatchSequencePattern()
                 typeof(Enumerable),
                 typeof(Queryable),
                  new[] {
-                     "ToLookup",
-                     "ToDictionary",
-                     "ToArray",
-                     "AsEnumerable",
-                     "ToList",
+                     nameof(Enumerable.ToLookup),
+                     nameof(Enumerable.ToDictionary),
+                     nameof(Enumerable.ToArray),
+                     nameof(Enumerable.AsEnumerable),
+                     nameof(Enumerable.ToList),
                      "Fold",
                      "LeftJoin",
-                     "Append",
-                     "Prepend",
-                     "ToHashSet"
+                     nameof(Enumerable.Append),
+                     nameof(Enumerable.Prepend),
+                     "ToHashSet",
+                     "SkipLast",
```

Comment:
Ideally (maybe we'll have to do it in phases, though I think it should be avoidable by having a P2P to queryable from system.linq), this shouldn't list SkipLast and TakeLast as queryable has those.